client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposureForm/ExposureForm.tsx

```diff
@@ -79,32 +79,22 @@ const ExposureForm = (props: any) => {
   }
 
   useEffect(() => {
+    const confirmedExposuresLogger = logger.setup({
```

Comment:
create confirmedExposuresLogger inside the else (you use this logger only there)